lib/kafka_ex/protocol.ex

```diff
@@ -37,6 +38,10 @@ defmodule KafkaEx.Protocol do
     @consumer_metadata_request
   end
 
+  defp api_key(:join_group) do
+    11
```

Comment:
I would use the module attribute `@join_group_request` here instead of the magic number 11